source/2019-04-05-the-ember-times-issue-92.md

```diff
@@ -29,6 +29,18 @@ Curious to learn more? Check out her post on Medium: [3 reasons why you should v
 
 ---
 
+## [Animations and Predictable Data Loading in Ember 🐹](https://crunchingnumbers.live/2019/04/02/animation-and-predictable-data-loading-in-ember/)
```

Comment:
@BradLeftley Thanks for writing up a summary of my article. I know it's a very long read, so I really appreciate your writing it on a short notice.

Could you change the section title to say `Animation` (singular instead of plural)? If you'd like to add yourself to the list of authors on the top of the Markdown file, that'd be great.